src/components/colorscale/calc.js

```diff
@@ -20,7 +20,7 @@ module.exports = function calc(trace, vals, containerStr, cLetter) {
 
     if(containerStr) {
         container = Lib.nestedProperty(trace, containerStr).get();
-        inputContainer = Lib.nestedProperty(trace._input, containerStr).get();
+        inputContainer = Lib.nestedProperty(trace, containerStr).get();
```

Comment:
@etpinard it was just a stop-gap. A few lines below the preexisting code set `cmin`/`cmax`/`colorscale` properties on the `inputContainer`. My understanding is that we not set or overwrite what the user gaveth us; that we'd decorate the output `container` with defaults (which was also in the code). I eliminated lines that mutated `inputContainer`, but glad to do something else if I'm trampling over some important aspect unknowingly.